1237-registrator_worklist/1238-creator_opens_my_messages.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Creator opens My Messages
+
+    @1238
+    Scenario Outline: Creator opens My Messages
+        Given that the user is logged in as Creator
+        When they click the menu item My Messages
+        Then they see the My Messages page
+        And they see a list of <RequestType> with fields:
+            | Publication title |
+            | Date              |
+        And they see that items' status is one of <RequestStatus>
+        And they see that each item in the list is expandable
+        Examples:
+            | RequestType | RequestStatus                 |
+            | DoiRequests | Approved, Rejected, Requested |
```

Comment:
Syns det er vanskelig å få dette til å gi helt mening... Lista over Meldinger vil på sikt kunne være en blanding av flere ulike typer: DoiRequest, ForApproval, Support, etc, hvor de kan ha ulike verdier for status.